guides/common/modules/con_defining-content-access-strategies-for-hosts.adoc

```diff
@@ -0,0 +1,106 @@
+. _Define composite content views_. You can use composite content views to combine and give hosts access to the content from multiple content views.
++
+For more information, see link:{ContentManagementDocURL}Creating_a_Composite_Content_View_content-management[Creating a composite content view] in _{ContentManagementDocTitle}_.
+. _Define architecture and operating system version restrictions._
+In custom products, you can set restrictions on the architecture and operating system versions for `{client-content-type}` repositories on which the product will be available.
+For example, if you restrict a custom repository to *{client-os} {client-os-major}*, it is only available on hosts running {client-os} {client-os-major}.
++
+ifdef::orcharhino[]
+ifdef::red_hat_enterprise_linux[]
+Red{nbsp}Hat repositories set architecture and operating system version restrictions automatically.
+endif::[]
+endif::[]
+ifndef::orcharhino[]
+Red{nbsp}Hat repositories set architecture and operating system version restrictions automatically.
+endif::[]
++
+[IMPORTANT]
+====
+Architecture and operating system version restrictions hold the highest priority among all other strategies.
+They cannot be overridden or invalidated by content overrides, changes to content views, or changes to lifecycle environments.
+For this reason, consider the other strategies before implementing architecture or operating system version restrictions.
+====
+. _Define release version restrictions._
+ifndef::orcharhino[]
+Certain Red{nbsp}Hat repositories, such as the {RHEL} dot release repositories, include a *Release version* in their repository metadata.
+If present, the release version is compared with the release version specified in the *System purpose* properties of the host.
+Based on this comparison, {Project} limits or restricts access to content.
++
+For more information about setting system purpose attributes, see {ManagingHostsDocURL}Creating_a_Host_managing-hosts[Creating a host in {ProjectName}] in _{ManagingHostsDocTitle}_.
+endif::[]
+ifdef::orcharhino[]
+ifdef::red_hat_enterprise_linux[]
+Certain Red{nbsp}Hat repositories, such as the {RHEL} dot release repositories, include a *Release version* in their repository metadata.
+If present, the release version is compared with the release version specified in the *System purpose* properties of the host.
+Based on this comparison, {Project} limits or restricts access to content.
++
+For more information about setting system purpose attributes, see {ManagingHostsDocURL}Creating_a_Host_managing-hosts[Creating a host in {ProjectName}] in _{ManagingHostsDocTitle}_.
```

Comment:
We're reducing the content of the _Creating a host_ procedure in #3443 because the generic procedure is not particulary useful without prerequisites from the Provisioning guide. Should we add a new procedure for setting system purpose attributes on a host instead?